junipervpn/vpn.py

```diff
@@ -325,6 +484,12 @@ def main():
                         help="Secondary password prefix")
     parser.add_argument('-o', '--oath', type=str,
                         help='OATH key for two factor authentication (hex)')
+    parser.add_argument('--connect-timeout', type=float, default=10,
+                        help='Timeout to wait between two connection attempts')
+    parser.add_argument('--ping-host', type=str,
+                        help='Host to ping to monitor the connection. Defaults to --host')
```

Comment:
Interestingly my VPN host only replies to `ping` packets from inside the VPN - so this doesn't work in ocproxy mode, and causes reconnects every 2s :(

Maybe we could leave the host unspecified, and enable this when someone adds something explicitly? (like `8.8.8.8`?). (or just use that as the default!).

Reply:
Updated the PR to have fallback ping hosts (in addition to user-defined ones)